src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -127,7 +128,43 @@ public void CreateTagger_should_return_null_when_analysis_is_not_supported()
         }
 
         [TestMethod]
-        public void CreateTagger_should_return_same_tagger_for_already_tracked_file()
```

Comment:
The GitHub web comparison makes it hard to see what's changed. I've inserted two new tests, and made tweaks to these two existing tests. See comments in the right-hand pane.